src/Bridge/Symfony/Resources/config/doctrine_orm.php

```diff
@@ -16,10 +16,11 @@
 use Sonata\Doctrine\Adapter\ORM\DoctrineORMAdapter;
 
 return static function (ContainerConfigurator $containerConfigurator): void {
+    // Use "service" function for creating references to services when dropping support for Symfony 4.4 (and add conflict)
     $containerConfigurator->services()
 
         ->set('sonata.doctrine.adapter.doctrine_orm', DoctrineORMAdapter::class)
             ->args([
-                service('doctrine'),
+                new ReferenceConfigurator('doctrine'),
```

Comment:
isn't there a `use` statement missing for this class?

Reply:
We're in the same namespace

Reply:
ah indeed :smile: missed that